### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by victoronline**
Added a line specifically referring to membership to help the clarity and flow of the document.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +31,20 @@ Learn more about [Working Groups](Working-Groups.md).
 * None yet.
 
 
+## The Planetary Software Organization Technical Steering Committee
+
+The Technical Steering Committee (TSC) is the governing body of the
+Planetary Software Organization, as described in its
+[Charter](TSC-Charter.md).
+
+The TSC primarily provides responsive input and organizational help
```
Comment (jlaura) [inline README.md:40]: What about - 'The TSC provides responsive input and organizational help to Top-Level Projects and Working Groups, as well as consultation to other interested software projects or endeavors.'

File: README.md [hunk 2]
```diff
@@ -12,18 +12,9 @@ manage the growth and evolution of their open source software by
...
-Planetary Software Organization, as described in its
-[Charter](TSC-Charter.md).
-
-The TSC primarily provides responsive input and organizational help
-to its Top-Level Projects and Working Groups, but is also available
-for consultation by other software projects or endeavors.
-
-Other governing documents for the TSC are located in the root
-folder of this repository.
+Software creators and maintainers of a project seeking to join The Planetary Software Organization can do so by applying for membership through the application process defined in [Project Lifecycle - Applying To Join](Project-Lifecycle.md#applying-to-join)
```
Comment (rbeyer) [inline README.md]: Due to recent merge, should now be `... through the [application process](Lifecycle.md#submitting-an-application).` or something like it with that link.

**Comment by jlaura** [comment]
Is this PR orphaned and we should just close or are we expecting updates pending comments?

**Comment by rbeyer** [comment]
I think this is still a good PR, but the comments need to be addressed.

**Review by victoronline — COMMENTED** [review]
This sounds good.

**Comment by jessemapel** [comment]
It looks like this has been updated, can we get this PR closed out soonish?

**Review by rbeyer — APPROVED** [review]

**Review by jessemapel — APPROVED** [review]

**Pull request merged**